src/Motorcycle/Chopper.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Motorcycle;
+
+class Chopper implements MotorcycleInterface
+{
+    use ShortInfo;
+
+    /**
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * @var string
+     */
+    protected $model;
+
+    /**
+     * @var string
+     */
+    protected $pilot;
+
+    /**
+     * @var int
+     */
+    protected $maxSpeed;
+
+    /**
+     * @return mixed
+     */
+    public function getName()
```

Comment:
I saw the same implementation in the class Sport. Maybe you can create AbstrcatMotorcycle class and move the common properties and getters-setters there.